enterprise/server/cmd/goinit/main.go

```diff
@@ -46,6 +45,8 @@ var (
 	setDefaultRoute         = flag.Bool("set_default_route", false, "If true, will set the default eth0 route to 192.168.246.1")
 	initDockerd             = flag.Bool("init_dockerd", false, "If true, init dockerd before accepting exec requests. Requires docker to be installed.")
 	gRPCMaxRecvMsgSizeBytes = flag.Int("grpc_max_recv_msg_size_bytes", 50000000, "Configures the max GRPC receive message size [bytes]")
+
+	isVMExec = flag.Bool("vmexec", false, "Whether to run as the vmexec server.")
```

Comment:
is this saving a bunch of space? Feels cleaner to call vmexec as a separate binary but maybe I'm missing why you did it like this.

Reply:
It increases the size of the initial RAM disk by 25 MB (it's currently 49 MB). I also figured it's slightly easier to manage flags, and share the common logging setup etc.

I agree that having it as a separate binary is probably a simpler mental model, though. I can make a separate `main` package for it, and then if we find the initrd size becomes a problem, we can have the executor write it directly to the scratch disk instead of having goinit copy it from from initrd to the scratch disk. wdyt?